### PR TITLE
Create systemconfig-snc.ini

### PR DESCRIPTION
**Pull request by DavidMi13**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Solutions/SAP/template/systemconfig-snc.ini [hunk 1]
```diff
@@ -0,0 +1,70 @@
+[Secrets Source]
+secrets = <DOCKER_RUNTIME/AZURE_KEY_VAULT/DOCKER_SECRETS/DOCKER_FIXED>
+keyvault = <SET_YOUR_AZURE_KEYVAULT>
+intprefix = <SET_YOUR_PREFIX>
+
+[ABAP Central Instance]
```
Comment (tamirkopitz) [inline Solutions/SAP/template/systemconfig-snc.ini:6]: The "user" and "passwd" fields are missing

**Review by tamirkopitz — CHANGES_REQUESTED** [review]

**Comment by NikTripathi** [comment]
@DavidMi13 Can you please resolve the comments. Thanks

**Comment by v-sabiraj** [comment]
@DavidMi13 Can you please resolve the requested changes. Thanks

**Comment by v-sabiraj** [comment]
@DavidMi13 , will you please resolve the requested changes. Thanks

**Comment by vmanojreddy** [comment]
@DavidMi13 , will you please resolve the requested changes. Thanks

**Comment by vmanojreddy** [comment]
@DavidMi13 , will you please resolve the requested changes. Thanks

**Comment by vmanojreddy** [comment]
Hey @DavidMi13 , we wanted to check on the status of this PR [Create systemconfig-snc.ini #3720](https://github.com/Azure/Azure-Sentinel/pull/3720). It is pending from more than 60 days. Let us know if any assistance is required for this PR. Per our standard operating procedures if no response is received in the next 7 business days we will close this PR. Thank you for your cooperation.

**Pull request closed without merging**